src/v7/transforms/route-ids.ts

```diff
@@ -0,0 +1,37 @@
+import { getLineEndings } from '../../util';
+
+export default function(file: any, api: any) {
+	let quote: string | undefined;
+
+	if (file.path && /routes\.ts(x)?$/.test(file.path)) {
```

Comment:
Do you think we should only do this for `src/routes.(ts|tsx)` not for any other route files in the code base? Also would the updated regex also process files like `src/mailroutes.tsx`?

Reply:
Yes I do, yes it would 🤦‍♂